### PR TITLE
Ownershipcard filter configurable

### PR DESCRIPTION
**Pull request by jvilimek**
## Hey, I just made a Pull Request!

Added `entityFilterKind` property for `EntityOwnershipCard`: Should allow change of filter for OwnershipCard. Default is Component, API. See issue #7806 

Where it affects UI:
image

Upated also example-app.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/org-ownershipcard-filteradded.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-"example-app": patch
-"@backstage/plugin-org": patch
+'example-app': patch
```
Comment (benjdlambert) [inline .changeset/org-ownershipcard-filteradded.md:2]: Don't need to bump `example-app` as that package is not published.

Comment (jvilimek) [reply .changeset/org-ownershipcard-filteradded.md]: ok, thanks. Fixed.

File: plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.test.tsx [hunk 2]
```diff
@@ -132,6 +133,17 @@ describe('OwnershipCard', () => {
       },
     );
 
+    expect(catalogApi.getEntities).toHaveBeenCalledWith({
```
Comment (jvilimek) [inline plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.test.tsx:136]: Original unit tests returned all entities no matter what the filter was. I left it this was. But I want to ensure the "backwards" Component&API kinds in filter. Is this the correct way? Or shall I implement the filter mock, e.g.

```typescript
catalogApi.getEntities.mockImplementation(
      (
        request?: CatalogEntitiesRequest,
        options?: CatalogRequestOptions,
      ): Promise<CatalogListResponse<Entity>> => {
        if (options === null) {
          throw 'options';
        }
        if (request === null) {
          throw 'request';
        }
        return Promise.resolve({
          items: items.filter(item => #TODO: filter item.kind based on request.kind ???,
            ),
          ),
        } as CatalogListResponse<Entity>);
      },
    );
```

Comment (jvilimek) [reply plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.test.tsx:136]: @benjdlambert WDYT?

Comment (benjdlambert) [reply plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.test.tsx:136]: Yeah let's add a test like that I think just to cover our bases. 👍

Comment (jvilimek) [reply plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.test.tsx:167]: Ok, done.

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
Nice work! Thansks!

**Pull request merged**